regimes/au/tax_categories.go

```diff
@@ -0,0 +1,111 @@
+package au
+
+import (
+	"time"
+
+	"github.com/invopop/gobl/cal"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/num"
+	"github.com/invopop/gobl/tax"
+)
+
+var taxCategories = []*tax.CategoryDef{
+	{
+		Code: tax.CategoryGST,
+		Name: i18n.String{
+			i18n.EN: "GST",
+		},
+		Title: i18n.String{
+			i18n.EN: "Goods and Services Tax",
+		},
+		Retained: false,
+		Rates: []*tax.RateDef{
+			{
+				Key: tax.RateExempt,
+				Name: i18n.String{
+					i18n.EN: "GST-free",
```

Comment:
I'd keep names simple:
```suggestion
					i18n.EN: "Exempt",
```